src/components/MessageList/__tests__/VirtualizedMessageList.test.js

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import { cleanup } from '@testing-library/react';
 import renderer from 'react-test-renderer';
+import './ResizeObserver.mock';
```

Comment:
should the file name end in .js?

Reply:
Actually named it like this deliberately because jest will run any file ending with .js and then complain that there are no tests in the file. Might be a little weird but this file is really only needed for jest